api/db/migrate/20170731155412_change_club_time_of_death_from_string_to_datetime.rb

```diff
@@ -0,0 +1,31 @@
+class ChangeClubTimeOfDeathFromStringToDatetime < ActiveRecord::Migration[5.0]
```

Comment:
General problem with this migration: It depends on the rest of our codebase. The convention which we use in the rest of our migrations is to avoid dependencies on things like ActiveRecord models and StreakClient wherever possible.

This is an example of removing StreakClient: https://github.com/hackclub/monolith/blob/master/api/db/migrate/20170417231434_add_stage_to_streakables.rb#L1

And here's a migration that doesn't use ActiveRecord: https://github.com/hackclub/monolith/blob/master/api/db/migrate/20170314102945_rename_hackbot_conversations_to_interactions.rb